includes/Core/Authentication/Exception/Google_Proxy_Code_Exception.php

```diff
@@ -22,6 +22,15 @@
  */
 class Google_Proxy_Code_Exception extends Exception {
 
+	/**
+	 * Temporary code for an undelegated proxy token.
+	 *
+	 * @since n.e.x.t Explicitly declared; previously, it was dynamically declared.
```

Comment:
We generally don't add descriptions for the version in which something was introduced. After that, we don't always document every single change with an added `@since`, we add these for significant changes (type, signature, etc). In this case it was missing entirely so `@since n.e.x.t` is all we should add. We don't need to document that it was previously a bug 😄